### PR TITLE
Service template initial

### PR DESCRIPTION
**Pull request by sevenzing**
I decided to add template for blockscout service using `cargo-template` :)

### CODE REVIEW

**Review by bragov4ik — APPROVED** [review]

File: service-template/.github/workflows/{{project-name}}.yml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        env:
+          POSTGRES_PASSWORD: admin
+          POSTGRES_USER: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
```
Comment (rimrakhimov) [inline service-template/.github/workflows/{{project-name}}.yml:39]: Can we setup the database initialization depending on whether the service needs to connect to postgresql database or not?

**Review by rimrakhimov — COMMENTED** [review]
Wow, finally we are here :+1: 

Some basic observations: 
1. Migration and entity crates are created even if we answered false on the corresponding questions
2. If logic crate has not been created, the migration and entity crates are also ignored. 
        What do you think about extracting logic and migration into the root directory - `{{project_name}}-migration` and `{{project_name}}-entity`?

File: service-template/{{project-name}}-proto/proto/v1/health.proto [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+// limitations under the License.
+
+// The canonical version of this proto can be found at
+// https://github.com/grpc/grpc-proto/blob/master/grpc/health/v1/health.proto
+
+option csharp_namespace = "Grpc.Health.V2";
+option go_package = "github.com/blockscout/blockscout-rs/{{project-name}}";
+option java_multiple_files = true;
+option java_outer_classname = "HealthProto";
+option java_package = "io.grpc.health.v2";
```
Comment (rimrakhimov) [inline service-template/{{project-name}}-proto/proto/v1/health.proto:26]: It should be the first version
```suggestion
option java_package = "io.grpc.health.v1";
```

File: service-template/{{project-name}}-proto/proto/v1/health.proto [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The canonical version of this proto can be found at
+// https://github.com/grpc/grpc-proto/blob/master/grpc/health/v1/health.proto
+
+option csharp_namespace = "Grpc.Health.V2";
```
Comment (rimrakhimov) [inline service-template/{{project-name}}-proto/proto/v1/health.proto:22]:
```suggestion
option csharp_namespace = "Grpc.Health.V1";
```

File: service-template/{{project-name}}-proto/proto/v1/{{project-name}}.proto [hunk 4]
```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package blockscout.{{projectName}}.v2;
```
Comment (rimrakhimov) [inline service-template/{{project-name}}-proto/proto/v1/{{project-name}}.proto:3]:
```suggestion
package blockscout.{{projectName}}.v1;
```

File: service-template/{{project-name}}-server/src/settings.rs [hunk 5]
```diff
@@ -0,0 +1,58 @@
+use blockscout_service_launcher::{
+    launcher::{ConfigSettings, MetricsSettings, ServerSettings},
+    tracing::{JaegerSettings, TracingSettings},
+};
+use serde::Deserialize;
+use std::collections::HashMap;
```
Comment (rimrakhimov) [inline service-template/{{project-name}}-server/src/settings.rs:6]: Is not needed

**Pull request merged**